MinMaxGame.java

```diff
@@ -1,99 +1,99 @@
 import java.util.Scanner;
 
 class MinMaxGame extends Game {
+	// Implements the negamax tree search algorithm.
 
 	public MinMaxGame(int height, int width, boolean playersTurn) {
 		super(height, width, playersTurn);
 	}
-	@override
+	@Override
 	protected int computerMoves() {
 		/* Computer moves. Afterwards returns
 		0 : Nobody won yet
 		2 : Computer won
 		3 : Table is full  */
```

Comment:
Rather than having it return numbers, you could declare constants some where that would represent these states and pass back those constants. It would make the code more readable.

Reply:
I will think about it!